.github/workflows/demos_visual_tests.yml

```diff
@@ -72,12 +72,13 @@ jobs:
       fail-fast: false
       matrix:
         STRATEGY: [screenshots, accessibility]
+        THEME: ['generic.light', 'material.blue.light', 'fluent.blue.light']
```

Comment:
Why we are testing only light themes?

Reply:
At the moment, yes. If there is a need to add other color schemes, we can do it later.